claim/claim.c

```diff
@@ -109,12 +116,35 @@ void claim_agent(char *claiming_arguments)
 #endif
 }
 
+#ifdef ENABLE_ACLK
+extern int aclk_connected, aclk_kill_link;
+extern void aclk_graceful_disconnect();
```

Comment:
Is `aclk_graceful_disconnect` actually used here?

Reply:
No, that was from the previous version